_posts/0000-01-02-cheysmith.md

```diff
@@ -0,0 +1,6 @@
+--
+layout: slide
+title: "Welcome to our second slide!"
+--
+Your text
+Use the left arrow to go back!
```

Comment:
https://www.google.com/imgres?imgurl=https%3A%2F%2Fi.imgflip.com%2Fc37k1.jpg&imgrefurl=https%3A%2F%2Fwww.akc.org%2Fexpert-advice%2Flifestyle%2Fhilarious-memes-for-dog-lovers%2F&tbnid=b_ybZ-TEXkNPHM&vet=12ahUKEwiptpnkht7yAhXgBFkFHWBpDMUQMygSegUIARDXAQ..i&docid=L0lsKHQRQPp2iM&w=490&h=274&q=dog%20meme&ved=2ahUKEwiptpnkht7yAhXgBFkFHWBpDMUQMygSegUIARDXAQ